__tests__/util/path.js

```diff
@@ -0,0 +1,45 @@
+/* @flow */
+
+jest.mock('../../src/util/user-home-dir.js', () => ({
+  default: '/home/foo',
+}));
+
+import {expandPath} from '../../src/util/path.js';
+
+describe('fileDatesEqual', () => {
+  const realPlatform = process.platform;
+
+  describe('!win32', () => {
+    beforeAll(() => {
+      process.platform = 'notWin32';
```

Comment:
There is a trick.
Jest runs tests on CI concurrently, even if in one file none of the tests are `test.concurrent`, CI will run tests suites in parallel.
And currently there is no elegant way to make sure that this test is running in isolation.

Double checking with @cpojer  that I am right.

How about a just running this test on non windows CI?
Yarn runs on OSX, linux and Windows, we'll notice if the test breaks